LICENSE

```diff
@@ -0,0 +1,375 @@
+This code is (c) by DEDIS/EPFL 2017 under the MPL v2 or later version.
```

Comment:
That's commonly used, e.g. the GPL is mostly used as "or any later version" - supposing mozilla will not screw up.